### DISCUSSION TITLE
Node v16.15.0 (npm v8.5.5) does not run postinstall

### DISCUSSION THREAD

**Discussion by marcomontalbano** [vercel/community#789] [Help] [answered]
Hello,
I was strugling with a weird error, when I realized that `npm install` was not running `postinstall` at all.
The setup is with NPM workspaces, where one of the workspace is a dependency of another and it contains a `postinstall` script.

I did a search and I found that there's a [known issue](https://github.com/npm/cli/issues/4804) on Node `v16.15.0` that is using npm `v8.5.5`. I changed the install command to `npm install -g npm && npm install` and application built succesfully.

Looks like `v16.15.0` is the latest that we can use when deploying a project on vercel, is it possible to upgrade the node version? Any plan for this?

I tested locally and I have the same issue. Updating to `v16.15.1` should be enough since it is using npm `v8.11.0` and this issue [has been solved](https://github.com/npm/cli/commit/5a50762faa37ae5964ae6f12595b20b367056c0a) in npm `v8.10.0`.

Thanks!

**Comment by swarnava** [comment]
Hi,

Thank you for reporting this. I am checking with the Build team and we will update you as soon as there is any progress.

**Comment by swarnava** [comment, marked as answer]
You can use corepack to use specific package manager: https://vercel.com/changelog/corepack-experimental-is-now-available

**Reply by marcomontalbano** [reply]
Adding the `ENABLE_EXPERIMENTAL_COREPACK=1` as environment variable, it works! Thanks!

Any plan to upgrade nodejs to a more recent version?

**Reply by swarnava** [reply]
Yes we will upgrade as soon as our upstream provider make it available. Unfortunately, we don't have an ETA on this.
